tests/noise_schedulers/test_variance_sampler.py

```diff
@@ -56,6 +64,41 @@ def expected_epsilons(self, expected_sigmas, noise_parameters):
 
         return torch.tensor(epsilons)
 
+    @pytest.fixture()
+    def expected_betas(self, expected_times, noise_parameters):
+        betas = []
+        for i in range(noise_parameters.total_time_steps):
+            betas.append(1.0 / (noise_parameters.total_time_steps - i))
+        return torch.tensor(betas)
+
+    @pytest.fixture()
+    def expected_alphas(self, expected_betas):
+        alphas = [1 - expected_betas[0]]
+        for beta in expected_betas[1:]:
+            alphas.append(alphas[-1] * (1 - beta.item()))
+        return torch.tensor(alphas)
+
+    @pytest.fixture()
+    def expected_q_matrix(self, expected_betas, num_classes):
+        expected_qs = []
+        for beta in expected_betas:
```

Comment:
This test is wrong. It is `1-beta` that should go on the diagonal, and `beta` in the last column.